lib/net.js

```diff
@@ -1276,12 +1276,15 @@ function listen(self, address, port, addressType, backlog, fd, exclusive) {
     return;
   }
 
+  // Master listens on port and cached, if workers use different backlog,
+  // the first one take effect.
```

Comment:
I meant a note in the reference documentation in doc/api, probably in cluster.markdown and net.markdown both. :-)

Let me know if you want some help with writing it.